krux_boto/__init__.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, logger = None, stats = None, parser = None):
         self.___parser = parser or get_parser(description = self.___name)
 
         ### in case we got some of the information via the CLI
-        self.___args    = self.___parser.parse_args()
+        self.___args    = self.___parser.parse_args(args = cli_args)
```

Comment:
Please remove the extra spaces before and after the equals sign. This should be done for you automatically if you use 'reformat code' in IntelliJ.